staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/promise/lockingpromise/lockingpromise.go

```diff
@@ -23,12 +23,12 @@ import (
 	"k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/promise"
 )
 
-// promisoid is the data and reading behavior common to all the
-// promise-like abstractions implemented here.  This implementation is
-// based on a condition variable.  This implementation tracks active
-// goroutines: the given counter is decremented for a goroutine
-// waiting for this varible to be set and incremented when such a
-// goroutine is unblocked.
+// promisoid is the data and behavior common to all the promise-like
+// abstractions implemented here.  This implementation is based on a
+// condition variable.  This implementation tracks active goroutines:
+// the given counter is decremented for a goroutine waiting for this
+// varible to be set and incremented when such a goroutine is
+// unblocked.
 type promisoid struct {
```

Comment:
BTW love the name `promisoid`-- very clear and short, but also seems to be newly coined, at least a search showed only typos in bad OCR files.  :)